### PR TITLE
adding foreman_api to non-scl comps

### PR DESCRIPTION
**Pull request by lzap**
SSIA

http://koji.katello.org/koji/taskinfo?taskID=41943

http://koji.katello.org/koji/taskinfo?taskID=41944

### CODE REVIEW

File: extras/packaging/rpm/rel-eng/comps/comps-foreman-fedora18.xml [hunk 1]
```diff
@@ -27,6 +27,7 @@
       <packagereq type="default">foreman-vmware</packagereq>
 
       <packagereq type="default">foreman-installer</packagereq>
+      <packagereq type="default">foreman-foreman_api</packagereq>
```
Comment (domcleal) [inline extras/packaging/rpm/rel-eng/comps/comps-foreman-fedora18.xml]: Um, surely `ruby193-rubygem-foreman_api`?

File: extras/packaging/rpm/rel-eng/comps/comps-foreman-rhel6.xml [hunk 2]
```diff
@@ -27,6 +27,7 @@
       <packagereq type="default">foreman-vmware</packagereq>
 
       <packagereq type="default">foreman-installer</packagereq>
+      <packagereq type="default">foreman-foreman_api</packagereq>
```
Comment (domcleal) [inline extras/packaging/rpm/rel-eng/comps/comps-foreman-rhel6.xml]: Surely `rubygem-foreman_api`?

Comment (lzap) [inline extras/packaging/rpm/rel-eng/comps/comps-foreman-rhel6.xml]: Ah I wanted to keep it in the installer "section" of the XML file, but indeed.

File: extras/packaging/rpm/rel-eng/comps/comps-foreman-fedora18.xml (hunk 1 above)
Comment (lzap) [inline extras/packaging/rpm/rel-eng/comps/comps-foreman-fedora18.xml]: Well, not in this case, but also `rubygem-foreman_api` :-)

Pushed

Comment (domcleal) [inline extras/packaging/rpm/rel-eng/comps/comps-foreman-fedora18.xml]: Oops, got the RHEL/Fedora files muddled.  You linked to ruby193-*, but I guess we don't need that for the installer here.

**Comment by domcleal** [comment]
Looks ok, do you have a redmine ticket?

**Comment by domcleal** [comment]
Merged as c41e2f4, thanks @lzap!

**Pull request closed without merging**